### ISSUE TITLE
att regions not found even though they exist

### ISSUE THREAD

**Issue by saifasghar** [HIT-ImmunologyLab/DBSCAN-SWA#28]
I'm encountering an issue where certain identified regions from the Phaster analysis of my sequence (available at https://phaster.ca/submissions/NC_019418) are not appearing in the output, despite being detected on the website.

Problem Description:
Upon reviewing the Phaster output for my sequence submission, I've noticed that att regions indicated are not displayed in the generated output.

Steps Taken:

I submitted the sequence using both FASTA and GenBank formats to ensure accuracy.
Reviewed the Phaster output thoroughly to confirm identified regions.
Ensured correct formatting and sequence submission.
Request for Assistance:
Could you please assist in addressing this discrepancy? The identified regions seem to be missing from the output, and I would appreciate guidance on how to resolve this issue.

[Streptococcus.phage.NJ2.txt](https://github.com/HIT-ImmunologyLab/DBSCAN-SWA/files/15369923/Streptococcus.phage.NJ2.txt)

I require the **functionality to be implemented within the code** to accommodate the analysis of multiple sequences. Would you be able to enhance the code to detect these regions as well? It is imperative that this functionality remains dynamic, ensuring accurate identification of att regions across different sequences.

Thank you for your support.

**Comment by HIT-ImmunologyLab** [comment]
@saifasghar 
Thank you for using DBSCAN-SWA!
(1) Based on the link you provided, the prophage region of NC_019418 detected by PHASTER is 260-35680, and we submitted the genbank file of NC_019418 downloaded from NCBI and the fasta file (Streptococcus. phage.NJ2.txt) you provided to DBSCAN-SWA for prediction, and the results obtained in the two formats are basically consistent with PHASTER. 
The results can be obtained by following the link: 
Genbank format: http://www.microbiome-bigdata.com/PHISDetector/index/tools/prophage/result/single_prophage_202442110104743213
Fasta format: http://www.microbiome-bigdata.com/PHISDetector/index/tools/prophage/result/single_prophage_202442110105228692
(2) DBSCAN-SWA supports the input of multi fasta format files.

**Comment by saifasghar** [comment]
Yes the results are matching that of phaster but I was talking about att regions.
Also can the functionality be implemented in the code itself because I have to research other DNA sequences too.
I am focused on att regions as of now and I cannot see any in the following links provided.
